tcf_website/static/css/site/pages/course_instructor.css

```
@@ -249,12 +249,17 @@
 }
 
 .rating-row__fill {
+  width: 0%;
   height: 100%;
   border-radius: var(--radius-full);
 
   transition: width var(--duration-normal) var(--ease-default);
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)